pl_examples/hydra_examples/conf/data/mnist.yaml

```diff
@@ -0,0 +1,8 @@
+# @package _group_
+
+batch_size: 5
+data_root: ./datasets
+num_workers: 4
+normalize:
+  - 0.5
+  - 0.2
```

Comment:
are there a corresponding dataclasses for configs in data?

Reply:
From my understanding this is up to the user in how their data is loaded and processed. There is no one PL way which is used to configure datasets. You do have to define a DataLoader at some point, which has some params.

Reply:
I see.
We can show an example for a couple of the more popular pytorch DataLoaders.

Reply:
Let me explain a bit further, torch comes packed with a default datasets here: https://pytorch.org/docs/stable/torchvision/datasets.html, which we can use to create a dataset. We also need to put this dataset into a dataloader which has its own definition here: https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader. We can parameterize both, should it be necessary.

Reply:
updated PR with an example